client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -71,7 +71,13 @@ function getHoldMessages( siteSlug, translate ) {
 			description: translate(
 				'Please check your email to confirm your address.'
 			),
-		}
+		},
+		EXCESSIVE_DISK_SPACE: {
+			title: translate( 'Excessive diskspace' ),
+			description: translate(
+				'This site cannot be transferred due to diskspace limitations.'
```

Comment:
I'm sure Editorial has better ideas, but how about: "Site Too Large: Media usage must be less than 50GB"